app/services/offender_service.rb

```diff
@@ -93,36 +36,15 @@ def self.get_multiple_offenders(offender_ids)
     }
   end
 
-  def self.get_multiple_offenders_as_hash(offender_ids)
-    get_multiple_offenders(offender_ids).map { |offender|
-      [offender.offender_no, offender]
-    }.to_h
-  end
-
-  def self.get_offenders_for_prison(prison)
-    OffenderEnumerator.new(prison).select { |offender|
-      offender.age >= 18 &&
-      (offender.sentenced? || offender.immigration_case?) &&
-      offender.criminal_sentence?
-    }
-  end
-
-  def self.get_unfiltered_offenders_for_prison(prison)
-    # Returns all offenders at the provided prison, and does not
-    # filter out under 18s or non-sentenced offenders in the same way
-    # that get_offenders_for_prison does.
-    OffenderEnumerator.new(prison)
-  end
-
   def self.get_sentence_details(booking_ids)
     Nomis::Elite2::OffenderApi.get_bulk_sentence_details(booking_ids)
   end
 
   # Takes a list of OffenderSummary or Offender objects, and returns them with their
   # allocated POM name set in :allocated_pom_name.
   # This is now only used by the SearchController.
-  def self.set_allocated_pom_name(offenders, caseload)
-    pom_names = PrisonOffenderManagerService.get_pom_names(caseload)
+  def self.set_allocated_pom_name(offenders, prison_id)
```

Comment:
Suspect this should be part of AllocationWithSentence. For later.... But can we pass in the prison rather than the id again?

Reply:
I've refactored AllocationWithSentence elsewhere, so this has somewhere to live now, so we can ignore this comment.